pi-apps-terminal-bash-edition.sh

```diff
@@ -232,11 +223,13 @@ while [ "$1" != "" ]; do
 		;;
 		search)
 			shift
+			arg="$*"
+			IFS=$'\n'
 			while read -r line; do
 				[[ -z "$line" ]] && continue
-				echo -e "${bold}${inverted}${light_blue}${underline}$line${normal}\n"
-				echo -e "${green}$(cat "$DIRECTORY/apps/$line/description" || echo "No description available")${normal}\n"
-			done < <(app_search "$1" 2>/dev/null)
+				echo -e "${underline}$line${normal}\n" | ack --passthru "$arg"
```

Comment:
The install script should install `ack`.

Reply:
I used `grep` to highlight text in bc6986b64e660395bd49795c64c581b82195af63.